src/scss/_global/_structure.scss

```diff
@@ -3,6 +3,8 @@ body {
   display: flex;
   flex-direction: column;
   justify-content: flex-start;
+  overflow-x: hidden;
+  overflow-y: scroll;
```

Comment:
This was a workaround to allow `<hr>` elements with the `divider` class applied to extend full-width despite having a more narrow parent div. 

We could place each horizontal rule that we want to be full width outside of any width tags, but then we would have to open and close those tags each time.

Is there a better way to do this?